assets/src/dashboard/components/button/index.js

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types'; // import styled from 'styled-components';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { BUTTON_TYPES } from '../../constants';
+
+const StyledButton = styled.button`
+  align-items: center;
+  background-color: ${({ theme, type }) =>
+    type === BUTTON_TYPES.PRIMARY ? theme.colors.bluePrimary : 'transparent'};
+  color: ${({ theme }) => theme.colors.white};
+  cursor: pointer;
+  border: 1px solid transparent;
+  border-radius: ${({ theme, type }) =>
+    type === BUTTON_TYPES.PRIMARY ? theme.border.buttonRadius : 'none'};
+  display: flex;
+  height: 40px;
+  min-width: 132px;
+  opacity: ${({ isCta }) => (isCta ? '1' : '0.75')};
+  padding: 0;
+  text-shadow: ${({ theme, type }) =>
+    type === BUTTON_TYPES.SECONDARY && theme.text.shadow};
+
+  &:focus,
+  &:active,
+  &:hover {
+    opacity: 1;
+    text-shadow: ${({ theme, type }) =>
```

Comment:
so no need for the default browser outline? Otherwise let's only activate the outline for keyboard users, see https://github.com/google/web-stories-wp/blob/66030e1ba87a7455bc4b76427203b21f334aec5a/assets/src/edit-story/components/library/panes/shared.js#L62

Reply:
I was following the button example in edit-story so thnk you for pointing out this shared file. I'll update this.

Reply:
We are going to use the util you created imported by the linked file. Our instruction was to, at this time, not have any cross importing. Once both apps are finished, I'm sure some optimization around a shared library can be looked at.

Reply:
Changes will now visible in storybook 
![keyboardonlyactiveborder](https://user-images.githubusercontent.com/10720454/76801170-59efdf00-6792-11ea-8145-8ed4d659a774.gif)